src/main/java/frc/robot/commands/TagPositionDrive.java

```diff
@@ -75,7 +75,16 @@ public void initialize() {
         // for safety, set command to null
         m_followTrajectory = null;
 
-        int centralTagId = m_vision.getCentralTagId();
+        // to keep track if target is substation or grid
+        boolean isSubstationTarget = false; //set target to grid first
+        // if robot is targeting a substation, set bool to true
+        if (m_targetPosition == Position.LEFT_SUBSTATION || m_targetPosition == Position.RIGHT_SUBSTATION) {
+            isSubstationTarget = true;
+        }
```

Comment:
The above tests will work, but it is a little cleaner to just assign the boolean directly from the if clause:
    boolean isSubstationTarget = (m_targetPosition == Position.LEFT_SUBSTATION || m_targetPosition == Position.RIGHT_SUBSTATION);